packages/playwright-core/browsers.json

```diff
@@ -48,8 +48,8 @@
       "revision": "1011",
       "installByDefault": true,
       "revisionOverrides": {
-        "mac12": "1010",
-        "mac12-arm64": "1010"
+        "mac12": "1011",
```

Comment:
whats the motivation for changing this?

Reply:
`1010` doesn't have `.tar.br`, and `1010` is identical to `1011` in functionality